ZAM_fnc_EZMLiteV2.1.1.sqf

```diff
@@ -6047,11 +6048,21 @@ MAZ_EZM_fnc_initFunction = {
 		_lines
 	};
 
-	MAZ_EZM_fnc_handleIntroCinematic = {
-		params ["_cinematicType","_backgroundSong","_intertitles", "_target"];
+	HYPER_EZM_fnc_handleIntroCinematic = {
+		params ["_cinematicType","_backgroundSong","_intertitles", "_zeusCanSeeCutscene", "_target"];
 
 		["Intro cinematic initiated for all players.","addItemOk"] call MAZ_EZM_fnc_systemMessage;
 
+		comment "if zeus shouldn't see the cutscene, we need to get the player set differenced with zeus player set";
+		private _allPlayers = [];
+		if(_zeusCanSeeCutscene) then {
+			comment "always prevent game mod from seeing the cutscene to avoid breaking their camera; only works if they don't have ezm running as well";
+			_allPlayers = allPlayers - allCurators;
```

Comment:
`allCurators` returns the Curator logic, not the unit assigned to the logic. 

You can get the Zeus players by doing:
`private _zeusPlayers = allCurators apply {getAssignedCuratorUnit _x};`

Some items in `_zeusPlayers` can be null, shouldn't be important in this case but for future reference.